src/scss/core/components/_qty-input.scss

```diff
@@ -1,33 +1,61 @@
 .quantity-button {
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: stretch;
+  min-width: 6rem;
+
   input {
-    max-width: 5rem;
+    z-index: 2;
+    min-width: 2rem;
+    max-width: 4rem;
+    height: 2.5rem;
+    padding: 0.375rem 0.75rem;
+    text-align: center;
+    border: 1px solid $input-border-color;
+    @include border-radius(0);
+
+    &::-webkit-outer-spin-button,
+    &::-webkit-inner-spin-button {
+      margin: 0;
+      appearance: none;
+    }
+
+    &[type="number"] {
+      appearance: textfield;
+    }
   }
 
   button {
     display: flex;
     align-items: center;
     justify-content: center;
-    height: 50%;
+    min-width: 2rem;
     padding: 0;
     border: 1px solid $input-border-color;
+    @include border-radius(0);
```

Comment:
All this code (and even more) is completely unnecesary as bootstrap already handles this by default, even with input height changed:
![obraz](https://user-images.githubusercontent.com/5007456/153615873-d361cc63-7fe7-40d5-ad19-654a9e4dea8e.png)

Reply:
It's related to material-icons inside spin buttons.